model/form-entities-table.js

```diff
@@ -269,12 +269,31 @@ module.exports = memoize(function (db) {
 				isEntitiesNestedMap = entityObjects.value instanceof this.database.NestedMap;
 				if (!isEntitiesNestedMap) return result;
 				Item = entityObjects.value.getItemType();
+				// FIXME: This is probably broken. foreach loop over dataForms isn't correct.
+				// FIXME: it should be iteration over dataForm.map. Look under toWSSchema method.
```

Comment:
Good catch. Open a ticket for that, please. Even better if you can fix it in scope of that, but only if it won't take more thatn 30 minutes (so in case of any complications just leave it).